### PR TITLE
WIP - Summary Section Styling

### PR DESCRIPTION
**Pull request by christianmemije**
## Summary

WIP - Summary Section Styling

### CODE REVIEW

File: kolibri/plugins/coach_tools/assets/src/vue/reports/summary-section/index.vue [hunk 1]
```diff
@@ -145,7 +177,24 @@
...
+  .summary-section-date
+    display: table-cell
+    padding: 10px
+    vertical-align: top
+
+  .summary-section-progress
+    width: 20%
+
+  .summary-section-date
+    width: 150px
```
Comment (indirectlylit) [inline kolibri/plugins/coach_tools/assets/src/vue/reports/summary-section/index.vue]: maybe you could pull out some of these values into a `reports.styl` file, and re-use them here and in the other vue file. E.g.

```
$progress-col-width = 20%
$data-col-width = 150px
```

etc

Comment (christianmemije) [reply kolibri/plugins/coach_tools/assets/src/vue/reports/summary-section/index.vue]: I was trying to require the file, but couldn't figure it out. How would I require that local stylesheet?

File: kolibri/plugins/coach_tools/assets/src/vue/reports/summary-section/index.vue [hunk 2]
```diff
@@ -3,64 +3,93 @@
...
-      <div>
+      <div class="summary-section-progress">
         <p>{{ $tr('content') }}</p>
         <progress-bar v-if="contentprogress !== undefined" :progress="contentprogress"></progress-bar>
-        <span v-else>{{ $tr('na') }}</span>
+        <span v-else><p>{{ $tr('na') }}</p></span>
+      </div>
+
+      <!-- TODO: WHY IS isRecentView INCORRECT?!-->
+      <div v-if="!isRecentView" class="summary-section-date">
```
Comment (indirectlylit) [inline kolibri/plugins/coach_tools/assets/src/vue/reports/summary-section/index.vue]: it's passed as a prop, so it's all lower-case: `isrecentview`

File: kolibri/plugins/coach_tools/assets/src/vue/reports/index.vue [hunk 3]
```diff
@@ -209,10 +211,12 @@
...
   @require '~kolibri.styles.coreTheme'
+  @require './reports.styl'
 
   .data-table
     width: 100%
     font-size: smaller
+    border-spacing: 0
 
     td, th
       padding: 10px
```
Comment (indirectlylit) [inline kolibri/plugins/coach_tools/assets/src/vue/reports/index.vue]: `$col-padding`

**Pull request merged**